src/Sarif/Autogenerated/Location.cs

```diff
@@ -45,11 +45,6 @@ public virtual SarifNodeKind SarifNodeKind
         [JsonProperty(DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate)]
         public virtual BigInteger Id { get; set; }
 
-        public bool ShouldSerializeId()
```

Comment:
>ShouldSerializeId [](http://example.com/codeflow?start=20&length=17)

Why did this get deleted? Is it because we specify -1 as a default value and 'emit default value' is false, so this is redundant?

Can you explain how this code got left behind, if so? Did we take an autogenerator fix?

Reply:
closing this, replied in the other one.